android/src/main/kotlin/br/com/oitiliveness3d/oiti_liveness3d/OitiLiveness3dPlugin.kt

```diff
@@ -84,9 +86,15 @@ class OitiLiveness3dPlugin : FlutterPlugin, MethodCallHandler, ActivityAware, Pl
         }
     }
 
-    private fun startLiveness3d(appKey: String?, environment: String?, textsBuilder: Map<String, String?>?, loadingAppearance: Map<String, Any?>?) {
+    private fun startLiveness3d(
+        appKey: String?,
+        environment: String?,
+        textsBuilder: Map<String, String?>?,
+        themeBuilder: Map<String, String?>?,
+        fontsBuilder: Map<String, String?>?,
+        loadingAppearance: Map<String, Any?>?) {
```

Comment:
Sugestão de indentação

```suggestion
    private fun startLiveness3d(
        appKey: String?,
        environment: String?,
        textsBuilder: Map<String, String?>?,
        themeBuilder: Map<String, String?>?,
        fontsBuilder: Map<String, String?>?,
        loadingAppearance: Map<String, Any?>?
    ) {
```